org.eclipse.winery.frontends/app/tosca-management/src/app/model/subMenuItem.ts

```diff
@@ -79,7 +79,7 @@ export class SubMenuItems {
     static readonly threatModeling: SubMenuItem = { displayName: 'Threat Modeling', urlFragment: 'threatmodeling' };
     static readonly topologyTemplate: SubMenuItem = { displayName: 'Topology Template', urlFragment: 'topologytemplate' };
     static readonly validSourcesAndTargets: SubMenuItem = { displayName: 'Valid Sources and Targets', urlFragment: 'validsourcesandtargets' };
-    static readonly capabilityTypeConstraints: SubMenuItem = { displayName: 'Constraints', urlFragment: 'constraints' };
+    static readonly capabilityTypeConstraints: SubMenuItem = { displayName: 'Valid Source Types', urlFragment: 'constraints' };
```

Comment:
Please check that this is only renamed for the YAML mode. Create a separate menu entry here if this is atm shared in the two modes.

Reply:
The variable is only used [here](https://github.com/eclipse/winery/blob/master/org.eclipse.winery.frontends/app/tosca-management/src/app/instance/instance.service.ts#L107). A check if YAML is used in done on line 105. Should be fine, imo